### DISCUSSION TITLE
thanhle.blog/chuyen-tro-thanh-software-engineer

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#57] [Ideas]
# thanhle.blog/chuyen-tro-thanh-software-engineer

Nghe về câu chuyện cuộc đời mình

https://thanhle.blog/blog/chuyen-tro-thanh-software-engineer

**Comment by nguyenhoaikhang37** [comment]
Tuổi thơ anh Thành dữ dội quá, hóng blog tiếp theo :))

**Reply by chungnq253** [reply]
+1 hóng p2 :D

**Comment by KyTran188** [comment]
hóng part 2 anh ơi
